src/physics/em/detail/UrbanMscScatter.hh

```diff
@@ -188,36 +176,12 @@ CELER_FUNCTION auto UrbanMscScatter::operator()(Engine& rng)
 
             CELER_ENSURE(std::abs(cth) <= 1);
 
-            UniformRealDistribution<real_type> sample_phi(0, 2 * constants::pi);
-            real_type                          phi = sample_phi(rng);
-            result.direction = rotate(from_spherical(cth, phi), inc_direction_);
-
-            // Displace the lateral position by the multiple scattering
-            if (input_.is_displaced && tau_ >= params_.tau_small)
-            {
-                real_type rmax2 = (true_path - geom_path)
-                                  * (true_path + geom_path);
-                if (rmax2 > 0)
-                {
-                    // Sample a unit direction of the displacement
-                    Real3 displacement = dir_displacement(rng, phi);
-
-                    // Rotate along the incident particle direction
-                    displacement = rotate(displacement, inc_direction_);
-
-                    // Scale with the lateral arm
-                    real_type arm = real_type(0.73) * std::sqrt(rmax2);
-                    for (auto i : range(3))
-                    {
-                        displacement[i] *= arm;
-                    }
-                    result.displacement = displacement;
-                }
-            }
+            real_type rmax2 = (true_path - geom_path) * (true_path + geom_path);
```

Comment:
Might be good to note that this is `ipow<2>(true_path) - ipow<2>(geom_path)`. I assume this form is more numerically stable?

Reply:
Okay. Changed it as suggested.